plop/templates/test.hbs

```diff
@@ -0,0 +1,22 @@
+import { submitResults } from '../../helpers';
+import { BaseTest } from '../../types';
+import { NightwatchBrowser, NightwatchCallbackResult } from 'nightwatch';
+
+
+const {{properCase name}}Test: BaseTest = {
+'@disabled': false,
+tags: [],
+developer: {{titleCase developer}},
+desiredCapabilities: {
+name: {{titleCase name}},
```

Comment:
is it possible to bundle the developer name in here too? it'll help Rosi with quickly identifying the person to go to